package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
@hasufell That sounds like something I wouldn't expect to find in a minor dot release. Adding new lint rules could easily be _not_ backwards compatible for some projects. In any event it's neither here nor there...

I was going to suggest Greenkeeper when I first saw this thread before I got down to where it was mentioned. That's the solution for "always having the very latest", not the changing the requirements. In this case I'd suggest keeping what `--save-dev` defaulted too.